builtin/game/chatcommands.lua

```diff
@@ -583,6 +583,10 @@ core.register_chatcommand("rollback_check", {
 			local name = puncher:get_player_name()
 			core.chat_send_player(name, "Checking " .. core.pos_to_string(pos) .. "...")
 			local actions = core.rollback_get_node_actions(pos, range, seconds, limit)
+			if actions == nil then
```

Comment:
1. This should be `if not actions then`.
  2. This should quick-fail as soon as the command is run if rollback recording is disabled.